modules/ROOT/pages/about-the-xml-configuration-file.adoc

```diff
@@ -0,0 +1,125 @@
+        xmlns:jms="http://www.mulesoft.org/schema/mule/jms"
+        xmlns:file="http://www.mulesoft.org/schema/mule/file"
+        xsi:schemaLocation="
+        http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/jms http://www.mulesoft.org/schema/mule/jms/current/mule-jms.xsd
+        http://www.mulesoft.org/schema/mule/file http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd">
+----
+
+[NOTE]
+Mule domains feature the `mule-domain` tag instead of `mule`.
+
+Be sure to specify all the necessary schema files. This can be time-consuming when setting up the configuration file by hand, but importing schema files provides the following time-saving benefits:
+
+* Auto-completion and context-specific help in your favorite IDE
+* Design-time configuration validation
+* Typed properties
+
+[NOTE]
+Anypoint Studio will add the required schemas as components are introduced.
+
+
+== Namespaces
+
+Each Mule module or connector has its own XML schema, including Mule core for it's community and enterprise versions. When you import a schema, it has its own namespace.
+
+To use the standard Mule elements, import the CE and EE namespaces:
+
+* http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+
+* http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+
+=== Default Namespace
+
+Typically, you set the Mule core schema as the default namespace for your configuration file. This means that any XML element without a prefix will come from the Mule core schema. To set the default namespace schema, specify `xmlns` immediately followed by the URL of the Mule schema, without the colon or namespace prefix you set in the previous example (e.g., `xmlns` instead of `xmlns:jms`):
+
+[source,xml,linenums]
+----
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+    xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+   ...config...
+</mule>
+----
+
+== Merging Configuration Files
+
+If you have multiple configuration files, you can import them into one configuration file so that you only have to specify one configuration.
+This is useful to extract connector configurations or other global elements. For example:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns=http://www.mulesoft.org/schema/mule/core ....>
+
+    <import file="global-prod-configurations.xml" />
+    <import file="global-error-handler.xml" />
+...
+----
+
+These imports can also be dynamic when combined with properties:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns=http://www.mulesoft.org/schema/mule/core ....>
+
+    <global-property name="env" value="dev"/>
+    <import file="global-${env}-configurations.xml" />
```

Comment:
I would add a not that the configuration properties for the import component can only by read from system properties, environment properties or the global-property component.

BTW: Review if there's already docs for <import/>. In such case just refer to that doc.

Reply:
I'll check whether the configuring properties doc in the see also includes that detail.